v4l2-sys/build.rs

```diff
@@ -4,18 +4,21 @@ use std::env;
 use std::path::PathBuf;
 
 fn main() {
-    let pkg_conf = pkg_config::Config::new()
+    #[cfg(not(target_os = "freebsd"))]
+    let extra_include_paths: Vec<String> = vec![];
+
+    #[cfg(target_os = "freebsd")]
+    let extra_include_paths: Vec<String> = pkg_config::Config::new()
         .probe("libv4l2")
```

Comment:
How does this sound:
```suggestion
        // `v4l2-sys` does not need to generate bindings against `libv4l2`, but utilizes this package to find the `videodev2.h` Linux UAPI header (provided by a compat package on FreeBSD?):
        .probe("libv4l2")
```

Reply:
No worries at all, thanks so much for bearing with me! I love taking something (a misunderstanding in this case) to the next level and make sure everyone is on the same page, followed by having correct and understandable code which is what you submitted here :+1: 

Thanks again for getting to the bottom of this!